plugins/janus_videoroom.c

```diff
@@ -2321,6 +2330,43 @@ void janus_videoroom_hangup_media(janus_plugin_session *handle) {
 	}
 }
 
+static void janus_videoroom_sdp_v_format_vp8(char *mline, int mline_size, int pt, int b, const char *video_mode) {
```

Comment:
I didn't know about that feature, either.  According to [this page](http://linux.die.net/man/3/printf), it is **not** part of the C99 standard but comes from the [Single Unix Specification](https://en.wikipedia.org/wiki/Single_UNIX_Specification).

I'm fine to switch all SDP templates to it because it would make using them simpler as long as there is no concern about portability.  As a caveat, all parameters have to use the positional notation and all parameters have to be referred to at least once.